cli/cmd/proxy/list/command.go

```diff
@@ -0,0 +1,218 @@
+package list
```

Comment:
This file is called `command` instead of `list` to match the pattern of the Consul-K8s control plane code. I am going to start shifting the code to this pattern so I can separate out "command" code from business logic in other commands down the line too.